onnxscript/tools/transformers_models/__init__.py

```diff
@@ -22,6 +27,38 @@ def has_transformers():
         return False
 
 
+def has_torch(version: str = ""):
```

Comment:
Just wondering: do these two functions belong in this module? I assume we have more flexible rules for tools/ but we need to communicate that these are not going to be stable apis.

Reply:
I can put them in onnxscript/testing. transformers and pytorch changes a lot. I expect some tests to only work with recent versions of both.

Reply:
Not suggesting onnxscript/testing because that's a set of more stable API we provide. I would consider `onnxscript._internal.testing`